package.json

```diff
@@ -62,7 +62,7 @@
   },
   "homepage": "https://github.com/ipfs/js-ipfs#readme",
   "devDependencies": {
-    "aegir": "^17.0.1",
+    "aegir": "^17.1.1",
```

Comment:
you can update `aegir` now to `18.0.3`

Reply:
I'm going to hold off on that one for this release and do that upgrade in 0.35